tools/ui-components/src/form-group/form-group.test.tsx

```diff
@@ -23,13 +26,20 @@ describe('<FormGroup>', () => {
     });
   });
 
-  it('provided controlId to label and control', () => {
+  it('should pass `controlId` to the label and input elements', () => {
     render(
-      <FormGroup controlId='my-control' data-testid='test-id'>
-        <FormControl aria-label='test' />
+      <FormGroup controlId='foo'>
+        <ControlLabel>Foo</ControlLabel>
+        <FormControl />
       </FormGroup>
     );
-    const input = screen.getByLabelText('test');
-    expect(input.id).toBe('my-control');
+
+    const label = screen.getByText('Foo');
+    const input = screen.getByLabelText('Foo');
+
+    expect(label).toBeInTheDocument();
+    expect(label).toHaveAttribute('for', 'foo');
+    expect(input).toBeInTheDocument();
+    expect(input).toHaveAttribute('id', 'foo');
```

Comment:
Should we add an assertion like: `expect(formGroup).not.toHaveAttribute('id', 'foo')`. It will prevent the same defect in the future